project/AppDependencies.scala

```diff
@@ -4,6 +4,8 @@ import sbt._
 
 object AppDependencies {
 
+  val silencerVersion = "1.7.1"
+
   val compile: Seq[ModuleID] = Seq(
     ws,
     "uk.gov.hmrc" %% "bootstrap-play-26" % "2.3.0",
```

Comment:
Don't forget to up the minor dependency versions (so long it doesn't break anything).